src/components/Select/Select.jsx

```diff
@@ -34,6 +36,8 @@ export default class Select extends React.Component {
     items: this.props.items || [],
     selectedItems: this.props.items || [],
     isFiltering: false,
+    searchValue: '',
+    isCustomItemFocused: false,
```

Comment:
Would be awesome to add some comments here to clarify what each prop in the state is being used for. Specifically, I can see us scratching our head a few months from now, wondering what `isCustomItemFocused` is all about 😂 . Curious how that feels @gomezjuliana ?

Reply:
hey @ivanaszuber how does this feel now?

Reply:
Feels great @gomezjuliana 👌 ! Thanks for adding extra context 🌹